### PR TITLE
update to latest version of winston

### PR DESCRIPTION
**Pull request by yads**
use async for better control flow

This is to enable new features in winston such as formatters.

### CODE REVIEW

File: lib/winston-redis.js [hunk 1]
```diff
@@ -83,44 +84,45 @@ Redis.prototype.log = function (level, msg, meta, callback) {
...
+    showLevel:   this.showLevel,
+    stringify:   this.stringify,
+    label:       this.label,
+    depth:       this.depth,
+    formatter:   this.formatter,
+    humanReadableUnhandledException: this.humanReadableUnhandledException
+  });
+
+  async.series([
+      function(cb) {
```
Comment (indexzero) [inline lib/winston-redis.js]: 2-space soft-tabs please.

**Comment by yads** [comment]
@indexzero spacing should be fixed now

**Comment by indexzero** [comment]
Thanks @yads! Any interest in being a new maintainer?

**Comment by yads** [comment]
@indexzero sure, my npm is yads

**Comment by indexzero** [comment]
Awesome. I've taken to the Mikeal Rogers approach of adding new maintainers: liberal on `git commit` access, and judicious on `npm` access. Once we've published out a few versions together I'll add you to the npm package. Until then, just ping me, @jcrugzz @pose, or @chjj whenever and we'll run `npm publish`

**Comment by yads** [comment]
Great, do you mind pushing one out now?

**Comment by indexzero** [comment]
Sure thing. Making it `1.0.0`. Semver from here on out.

**Pull request merged**